py/src/ai/h2o/sparkling/ml/params/H2OAutoMLParams.py

```diff
@@ -17,253 +17,20 @@
-        return self.getOrDefault(self.maxAfterBalanceSize)
-
-    def getKeepCrossValidationPredictions(self):
-        return self.getOrDefault(self.keepCrossValidationPredictions)
-
-    def getKeepCrossValidationModels(self):
-        return self.getOrDefault(self.keepCrossValidationModels)
-
-    def getKeepCrossValidationFoldAssignment(self):
-        return self.getOrDefault(self.keepCrossValidationFoldAssignment)
-
-
-    def getMaxModels(self):
-        return self.getOrDefault(self.maxModels)
-
-    def getMaxRuntimeSecsPerModel(self):
-        return self.getOrDefault(self.maxRuntimeSecsPerModel)
-
-    def getExportCheckpointsDir(self):
-        return self.getOrDefault(self.exportCheckpointsDir)
-
-    def getExploitationRatio(self):
-        return self.getOrDefault(self.exploitationRatio)
-
-    ##
-    # Setters
-    ##
-    def setIgnoredCols(self, value):
-        return self._set(ignoredCols=value)
-
-    def setTryMutations(self, value):
-        return self._set(tryMutations=value)
-
-    def setIncludeAlgos(self, value):
-        return self._set(includeAlgos=value)
-
-    def setExcludeAlgos(self, value):
-        return self._set(excludeAlgos=value)
-
-    def setProjectName(self, value):
-        return self._set(projectName=value)
-
-    def setMaxRuntimeSecs(self, value):
-        return self._set(maxRuntimeSecs=value)
-
-    def setStoppingRounds(self, value):
-        return self._set(stoppingRounds=value)
-
-    def setStoppingTolerance(self, value):
-        return self._set(stoppingTolerance=value)
-
-    def setStoppingMetric(self, value):
-        return self._set(stoppingMetric=value)
-
-    def setSortMetric(self, value):
-        return self._set(sortMetric=value)
-
-    def setBalanceClasses(self, value):
-        return self._set(balanceClasses=value)
-
-    def setClassSamplingFactors(self, value):
-        return self._set(classSamplingFactors=value)
-
-    def setMaxAfterBalanceSize(self, value):
-        return self._set(maxAfterBalanceSize=value)
-
-    def setKeepCrossValidationPredictions(self, value):
-        return self._set(keepCrossValidationPredictions=value)
-
-    def setKeepCrossValidationModels(self, value):
-        return self._set(keepCrossValidationModels=value)
-
-    def setKeepCrossValidationFoldAssignment(self, value):
-        return self._set(keepCrossValidationFoldAssignment=value)
-
-    def setMaxModels(self, value):
-        return self._set(maxModels=value)
-
-    def setMaxRuntimeSecsPerModel(self, value):
-        return self._set(maxRuntimeSecsPerModel=value)
-
-    def setExportCheckpointsDir(self, value):
-        return self._set(exportCheckpointsDir=value)
-
-    def setExploitationRatio(self, value):
-        return self._set(exploitationRatio=value)
+class H2OAutoMLParams(
+    H2OCommonParams,
+    H2OAutoMLBuildControlParams,
+    H2OAutoMLBuildModelsParams,
+    H2OAutoMLInputParams,
+    H2OAutoMLStoppingCriteriaParams,
+    HasMonotoneConstraints
+):
```

Comment:
nit: `):` could be on previous line

Reply:
python seems to be ok with the proposal, but Idea starts complaining: `PEP8: continuation line with same indent as next logical line` If this warning is unimportant, I can do it. WDYT?

Reply:
Ah, cool. Didn't know about PEP8. Let's keep it as it is :)